src/SprykerEco/Zed/Econda/Communication/Plugin/ProductsPlugin.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+/**
+ * MIT License
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerEco\Zed\Econda\Communication\Plugin;
+
+use Generated\Shared\Transfer\BatchResultTransfer;
+use Generated\Shared\Transfer\LocaleTransfer;
+use Spryker\Zed\Kernel\Communication\AbstractPlugin;
+use SprykerEco\Zed\Econda\Business\Exporter\Writer\WriterInterface;
+use SprykerEco\Zed\Econda\Dependency\Plugin\EcondaPluginInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ * @method \SprykerEco\Zed\Econda\Business\EcondaFacade getFacade()
+ * @method \SprykerEco\Zed\Econda\EcondaConfig getConfig()
+ */
+class ProductsPlugin extends AbstractPlugin implements EcondaPluginInterface
+{
+    /**
+     * @param \Generated\Shared\Transfer\LocaleTransfer $locale
+     * @param \Generated\Shared\Transfer\BatchResultTransfer $result
```

Comment:
Naming suggestions: `$localeTransfer` and `$batchResultTransfer`

Reply:
Done